universal-login-commons/test/core/utils/deepCopy.ts

```diff
@@ -1,5 +1,5 @@
 import {expect} from 'chai';
-import {deepCopy} from '../../../lib/core/utils/deepCopy';
+const deepCopy = require('lodash.clonedeep');
 
 describe('deepCopy', () => {
```

Comment:
Do we need to test the imported library?

Reply:
We can now remove that file.